### DISCUSSION TITLE
DS Emulator on Android that uses dsv files

### DISCUSSION THREAD

**Discussion by alesso425** [rafaelvcaetano/melonDS-android#1335] [Support]
Hello.

I recently added the liberty ticket mystery gift to a Pokémon Black dsv file through an application on my computer. The dsv file opens and works flawlessly on Desmume on the pc, but when I transfer the file over to my phone (into the save folder in the designated emulator storage), the emulator cannot find the dsv and on "New Game" pops up instead the existing game save.

Any advice would help out greatly!

**Comment by foolishgrunt** [comment]
Late, but for future reference...

MelonDS only uses .sav files in raw format. If the .dsv file was created by drastic, or desmume, it is probably a raw file, but with useless stuff appended to the end. You can rename to .dsv file to .sav, then use a hex editor to trim the useless stuff.

For for information:
https://melonds.kuribo64.net/board/thread.php?pid=492#492
